src/redux/actions/ratingAction.js

```diff
@@ -0,0 +1,44 @@
+import * as types from '../actionTypes/ratesActionTypes';
+import apiCall from '../../helpers/apiCall';
+import { notificationSuccess, notificationError } from '../../helpers/notificationPopUp';
+
+export const inputData = (type, data) => ({
+  type,
+  payload: data,
+});
+
+export const onEditInputAction = (data) => async (dispatch) => {
+  dispatch(inputData(types.ADD_ACCOMMODATION_RATE_INPUT, data));
+};
+
+export const submitRatingAction = (data) => async (dispatch) => {
+  const Option = {
+    headers: {
+      token: localStorage.getItem('token'),
+    },
+  };
+  const { starRateInput, feedbackInput } = data;
+  const submitRating = {
+    rating: starRateInput,
+    review: feedbackInput,
+    accommodationId: 59,
+  };
+
```

Comment:
@NiyongaboEric  Kindly, would you get me to know how this `accommodation id` is going to be taken in case you want to rate another accommodation?

Reply:
My task depend on another task. I am expecting to get Id from viewing all bookings page. If a user clicks on the rating button of a given booking, I will keep the accommodation ID. 

The image I shared below is how I am planning to achieve it.

<img width="1174" alt="Screen Shot 2020-02-05 at 10 35 26" src="https://user-images.githubusercontent.com/33821734/73825266-56b51b00-4804-11ea-8855-fa88605138cf.png">